### DISCUSSION TITLE
by passsing passwd

### DISCUSSION THREAD

**Discussion by baccioni** [dnschneid/crouton#4951] [Q&A]
hey fellas, im a new bee literally i be having some difficulties setting up linux on my chromeos can someone help me out
![Screenshot 2023-03-21 23 09 52](https://user-images.githubusercontent.com/128552230/226765001-24d943c1-8435-49cd-aa3e-cab077022e2f.png)

**Comment by Taxrosdev** [comment]
have you seen/ran this command before? ```chromeos-setdevpasswd```?

Unsure if running it again will help you unset the password, but it is likely you set one.
